pages/test_analytics/ruby_collectors.md.erb

```diff
@@ -47,6 +47,56 @@ Before you start, make sure RSpec runs with access to [CI environment variables]
     $ git push
     ```
 
+5. Make sure that the Test Analytics [environment variables](#environment-variables) are set so that the collector can use them in your Test Analytics dashboard.
+
+Once you're done, in your Test Analytics dashboard, you'll see analytics of test executions on all branches that include this code.
+
+If you don't see branch names, build numbers, or commit hashes in the Test Analytics UI, then read [CI environments](/docs/test-analytics/ci-environments) to learn more about exporting your environment to the RSpec collector.
+
+## Minitest collector
+
+[Minitest](https://github.com/minitest/minitest) provides a complete suite of testing facilities supporting TDD, BDD, mocking, and benchmarking.
+
+If you're already using Minitest for your tests, add the `buildkite-collector` gem to your code to collect your test results into your Test Analytics dashboard.
+
+1. Create a new branch:
+
+    ```
+    git checkout -b install-buildkite-collector
+    ```
+
+2. Add `buildkite-collector` to your `Gemfile` in the `:test` group:
+
+    ```rb
+    group :test do
+      gem "buildkite-collector"
+    end
+    ```
+
+3. Run `bundle` to install the gem and update your `Gemfile.lock`:
+
+    ```sh
+    $ bundle
```

Comment:
Let’s remove all dollar sign for easier copy-paste.

```suggestion
    bundle
```

Reply:
This is a different issue, we should really be showing the `$` and making it not copy-pastable from the code block language, but that is a problem for another day.